nf_core/modules/test_yml_builder.py

```diff
@@ -25,6 +25,7 @@
 import nf_core.utils
 from nf_core.components.components_command import ComponentCommand
 
+from ..lint_utils import run_prettier_on_file
```

Comment:
maybe we should move this function into `utils` now that we use it more widespread?

Reply:
We talked about incorporating this into the JSON / YAML helper functions that dump things to files. Off the top of my head they're in the top level utils (?) so that would be my preferred location as well.

Reply:
That's fine with me. But the issue of where what lives wasn't introduced here.

Resolving this as off-topic.